src/main/java/xyz/jpenilla/squaremap/client/gui/screen/widget/MiniMapWidget.java

```diff
@@ -34,7 +34,11 @@ public int getHeight() {
 
     @Override
     public void renderButton(PoseStack matrixStack, int mouseX, int mouseY, float delta) {
+        matrixStack.pushPose();
```

Comment:
How is this diff related to the rest of the changes/why is it needed? Reverting it doesn't seem to have any obvious effect.